packages/commerce-sdk-react/src/types.ts

```diff
@@ -0,0 +1,24 @@
+interface CommonHookResponse {
+    error: Error | undefined
+    isLoading: boolean
+}
+
+// These are the common params for all query hooks
+// it allows user to override configs for specific query
+interface QueryParams {
+    siteId?: string
+    locale?: string
+    currency?: string
+    organizationId?: string
+    shortCode?: string
+}
```

Comment:
How do we know these params are the only common ones to all queries?

Reply:
Speaking of `currency` and `locale`: I saw this comment in the retail-react-app

https://github.com/SalesforceCommerceCloud/pwa-kit/blob/54bd7b5b9538a112a09d0fab2326e0cf5ef2dd74/packages/template-retail-react-app/app/commerce-api/index.js#L122-L124

So not all API endpoints expects to see currency or locale.

But I think here the meaning of "common params" is more like: out of all the _possible_ params, these are the common ones.